addOns/automation/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+- Added Import Job profile
```

Comment:
```
### Added.
- Import Job profile (Issue 7078).
```

Similar for the other changelog. I'll review the rest later.

Reply:
Done. Please check